fog/ledger/server/src/key_image_service.rs

```diff
@@ -147,30 +149,42 @@ impl<L: Ledger + Clone, E: LedgerEnclaveProxy> KeyImageService<L, E> {
         queries: Vec<attest::NonceMessage>,
     ) -> MultiKeyImageStoreResponse {
         let mut response = MultiKeyImageStoreResponse::new();
+        let uri = fog_ledger_store_uri;
         // The router needs our own URI, in case auth fails / hasn't been started yet.
-        response.set_store_uri(fog_ledger_store_uri.url().to_string());
+        response.set_store_uri(uri.url().to_string());
         // Default status of AUTHENTICATION_ERROR in case of empty queries
         response.set_status(MultiKeyImageStoreResponseStatus::AUTHENTICATION_ERROR);
 
+        let start_time = Instant::now();
+        let subdomain = uri.subdomain().unwrap_or("");
```

Comment:
Is there a real advantage to putting the full hostname? That's what we've done pretty much everywhere else metrics reference a hostname.